### DISCUSSION TITLE
Error at Component OpenAI with seed and Model kwargs

### DISCUSSION THREAD

**Discussion by Schinter** [langflow-ai/langflow#2865] [General]
Please fix the errors

Error building Component OpenAI: 1 validation error for ChatOpenAI root Parameters {'seed'} should be specified explicitly. Instead they were passed in as part of model_kwargs parameter. (type=value_error)

**Comment by marcosbeto** [comment]
Same error here. No idea how to fix it.

**Comment by ragingbal** [comment]
Most likely due to API changes. Temporary fix.

* Click on the code icon of the OpenAI model node.
* Comment out the line " model_kwargs["seed"] = seed" ( line 96 in my version)
* Add seed like so below the line "temperature=temperature or 0.1," ( line 108) - "seed = seed,"
Hope explanation was sufficient

**Reply by taylorbayouth** [reply]
This worked for me. However, now I get this error:
 TypeError: Expected mapping type as input to ChatPromptTemplate. Received <class 'list'>.

**Reply by taylorbayouth** [reply]
I think the reason why I'm receiving this new error is because of another change in the component. I changed "advanced=True" to "advanced=False" to get the system message working. While this does expose the field, it gives me the mapping error in my previous message.

        FloatInput(name="temperature", display_name="Temperature", value=0.1),
        BoolInput(name="stream", display_name="Stream", info=STREAM_INFO_TEXT, advanced=True),
        StrInput(
            name="system_message",
            display_name="System Message",
            info="System message to pass to the model.",
            advanced=False,
        ),
